realm-helpers/src/main/java/chat/rocket/android/realm_helper/RealmObjectObserver.java

```diff
@@ -22,7 +22,7 @@ protected T extractObjectFromResults(RealmResults<T> results) {
     }
 
     protected boolean isSame(T element1, T element2) {
-      return false;
+      return element1.equals(element2);
```

Comment:
I'm not sure, but I suspect `element1.equals(element2)` is always false.
I will check if so.